<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewItemDetails</class>
 <widget class="QDialog" name="NewItemDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>431</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>19</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="17" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="TouchButton" name="accept_button">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #31363B;
border: 0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resource_BY.qrc">
         <normaloff>:/Icons/Icons/GreenCheckIcon_Finished.png</normaloff>:/Icons/Icons/GreenCheckIcon_Finished.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TouchButton" name="cancel_button">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #31363B;
border: 0;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Resource_BY.qrc">
         <normaloff>:/Icons/Icons/RedCancelIcon_Finished.png</normaloff>:/Icons/Icons/RedCancelIcon_Finished.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="17" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_optional">
     <property name="font">
      <font>
       <family>Cronus Round</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Optional Details</string>
     </property>
     <widget class="QLabel" name="exp_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Expiration Date</string>
      </property>
     </widget>
     <widget class="QComboBox" name="month_combo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="monthlabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Month</string>
      </property>
     </widget>
     <widget class="QLabel" name="day_label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Day</string>
      </property>
     </widget>
     <widget class="QLabel" name="year_label">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Year</string>
      </property>
     </widget>
     <widget class="QComboBox" name="day_combo">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="year_combo">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>124</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="favorites_check">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add to Favorites List</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_main">
     <property name="font">
      <font>
       <family>Cronus Round</family>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="title">
      <string>New Item Details</string>
     </property>
     <widget class="QLineEdit" name="itemName_textBox">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>311</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="itemName_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Item Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="category_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Product Category</string>
      </property>
     </widget>
     <widget class="QComboBox" name="itemQty_combo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>81</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="itemQty_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Item Quantity</string>
      </property>
     </widget>
     <widget class="QComboBox" name="category_combo">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>254</width>
        <height>44</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="pkgQty_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>130</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>19</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quantity Per Package</string>
      </property>
     </widget>
     <widget class="QComboBox" name="pkgQty_combo">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cronus Round</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TouchButton</class>
   <extends>QPushButton</extends>
   <header>Widgets/touchButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../QDarkStylesheet/style.qrc"/>
  <include location="../Resource_BY.qrc"/>
 </resources>
 <connections/>
</ui>
